pootle/static/js/browser/components/Stats.js

```diff
@@ -45,12 +47,19 @@ const Stats = React.createClass({
   },
 
   render() {
+    if (!this.state.topContributors.length) {
```

Comment:
This way we are not rendering the "Top Contributors for the Last 30 Days" header. Not sure if that is the best idea.